lighthouse-core/gather/driver.js

```diff
@@ -150,7 +150,11 @@ class Driver {
         returnByValue: true
       }).then(result => {
         clearTimeout(asyncTimeout);
-        resolve(result.result.value);
```

Comment:
maybe i'll just file an issue for now and can address in a separate PR in depth?